exporter/elasticsearchexporter/utils_test.go

```diff
@@ -50,7 +50,7 @@ type httpTestError struct {
 	cause   error
 }
 
-const currentESVersion = "8.4.0"
+const currentESVersion = "7.17.7"
```

Comment:
Does this PR still support v8 server? If so can you please update the tests to test again both versions?

Reply:
@dmitryax  sorry for late response. yes it does. I have tested agains both of each very first and the most current version. the result was fine as expected.